vendor/arm/pmu/pmu_profiler.cpp

```diff
@@ -49,11 +49,11 @@ PmuProfiler::PmuProfiler(const CpuCounterSet &enabled_counters) :
 			try
 			{
 				// Create a PMU counter with the specified configuration
-				auto pmu_counter_res = pmu_counters_.emplace(counter, PmuCounter{pmu_config->second});
+				auto pmu_counter_res = pmu_counters_.emplace(counter, pmu_config->second);
```

Comment:
This reverts https://github.com/ARM-software/HWCPipe/pull/19 , was it intentional?

Reply:
Yep, I found out that #19 caused a regression on #17, so this solution works for both.